docs/guide_running_portal_locally.md

```diff
@@ -1,4 +1,19 @@
-# Guide to running portal locally with a custom portal config
+# Guide to running data-portal locally
+
+This guide describes how to test data-portal code changes or configuration changes locally.
+
+## Instructions for internal or external use
+
+1. Your Data Commons should be running and accessible, with a running `data-portal` pod.
+2. Clone the `data-portal` repo and `cd` to it.
+3. Run `npm i`.
```

Comment:
should we advocate on using `npm ci` in here?

Reply:
i updated that everywhere i found `npm install`